src/plugins/plot/res/templates/plot-options-edit.html

```diff
@@ -46,12 +46,13 @@
                             <label>Value:</label>
                             <span class="control">
                                 <div class="select">
-                                    <select ng-model="form.series[$index].yKey">
-                                        <option ng-repeat="option in form.series[$index].yAxisOptions"
+                                    <select ng-model="seriesForm.yKey">
+                                        <option ng-repeat="option in seriesForm.yAxisOptions"
                                                 value="{{option.value}}"
-                                                ng-selected="option.value == form.series[$index].yKey">
+                                                ng-selected="option.value === seriesForm.yKey">
```

Comment:
A less strict equal here could be of use if the data somehow changes value, but strict equal is 👍